src/v1/profile/LoginBottomSheet.tsx

```diff
@@ -0,0 +1,50 @@
+import { IconClose, IconKakao, LogoWithText } from '@public/icons';
+
+import Button from '@/v1/base/Button';
+import BottomSheet from '@/v1/base/BottomSheet';
+
+type LoginBottomSheetProps = {
+  isOpen: boolean;
+  onClose: () => void;
+};
+
+/**
+ * @todo
+ * client_id를 활용한 카카오 로그인 페이지 이동
+ */
+
+const LoginBottomSheet = ({ isOpen, onClose }: LoginBottomSheetProps) => {
+  const handleClickKakaoLogin = () => {
+    return (location.href = `${process.env.NEXT_PUBLIC_API_URL}/oauth2/authorize/kakao?redirect_uri=${process.env.NEXT_PUBLIC_CLIENT_REDIRECT_URI}`);
```

Comment:
comment;

```suggestion
    return (location.href = `${process.env.NEXT_PUBLIC_KAKAO_LOGIN_URL}`);
```

페이지 이동 href를 
`https://kauth.kakao.com/oauth/authorize?client_id=${Client_Id}&redirect_uri=${redirect_uri}&response_type=code`
으로 수정할 예정입니다.

위와같은 방법을 쓰면 Kakao OAuth의 client_id 노출이 불가피합니다

client_id가 노출되어도 되는가에 대한 아티클입니다
- [아티클1](https://devtalk.kakao.com/t/rest-api-api-key/80122)
- [아티클2](https://www.inflearn.com/questions/642156/%EC%B9%B4%EC%B9%B4%EC%98%A4%EB%A1%9C%EA%B7%B8%EC%9D%B8-%EC%8B%9C-%ED%94%84%EB%A1%A0%ED%8A%B8%EB%8B%A8%EC%97%90-%EC%9E%88%EB%8A%94-client-id%EB%8A%94-%EC%96%B4%EB%96%BB%EA%B2%8C-%EC%88%A8%EA%B8%B0%EB%82%98%EC%9A%94)
- [아티클3](https://devtalk.kakao.com/t/rest-api/128734)